File: instalacao-ci/application/models/Entity/Moderador.java
package br.edu.unipampa.app.domain;

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;

/**
 * Created by Esther Favero on 15/05/2018.
 */
@Entity(name = "moderadores")
public class Moderador {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;

    private String nome;

    public Long getId() {
        return id;
    }

    public String getNome() {
        return nome;
    }

    public void setNome(String nome) {
        if(nome == null){
            throw new IllegalArgumentException();
        } else {
            this.nome = nome;
        }
    }
}

File: instalacao-ci/application/models/Entity/Moderador.php
<?php

namespace Entity;


use Doctrine\Common\Collections\ArrayCollection;


/**
 * @Entity
 * @Table(name="moderadores")
 */
class Moderador
{

    /**
     * @Id
     * @Column(type="integer", nullable=false)
     * @GeneratedValue(strategy="IDENTITY")
     */
    private $id;

    /**
     * @Column(type="string", nullable=false)
     */
    private $nome;

    /**
     * @OneToMany(targetEntity="Entity\Reuniao", mappedBy="moderador")
     */
    private $reunioes;

    /**
     * Moderador constructor.
     */
    public function __construct()
    {
        $this->reunioes = new ArrayCollection();
    }

    /**
     * @return mixed
     */
    public function getId()
    {
        return $this->id;
    }

    /**
     * @param mixed $id
     */
    public function setId($id)
    {
        $this->id = $id;
    }

    /**
     * @return mixed
     */
    public function getNome()
    {
        return $this->nome;
    }

    /**
     * @param mixed $nome
     */
    public function setNome($nome)
    {
        $this->nome = $nome;
    }

    /**
     * @return mixed
     */
    public function getReunioes()
    {
        return $this->reunioes;
    }

    /**
     * @param mixed $reunioes
     */
    public function setReunioes($reunioes)
    {
        $this->reunioes = $reunioes;
    }

    /**
     * @param Reuniao $reuniao
     */
    public function abrirReuniao(Reuniao $reuniao)
    {
        $reuniao->setEstaAberta(true);
    }

}
